### ISSUE TITLE
Create and update tree nodes

### ISSUE THREAD

**Issue by mzuccara** [brimdata/react-arborist#180]
Is there an uncontrolled way (with tree api ref) to modify a subset of properties of a node (and not only the name)?
The same for a creation of a node, providing custom data.

**Comment by jameskerr** [comment]
I don't believe so. You'll need to provide the data prop and implement the data modification handlers. However, there is some code already written that you can use as inspiration here.  https://github.com/brimdata/react-arborist/pull/172/files
